tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java

```diff
@@ -1452,11 +416,10 @@ private static void runOnSchedule(Class<?> initClazz, BLangIdentifier name, Sche
             final Throwable t = out.panic;
             if (t != null) {
                 if (t instanceof BLangRuntimeException) {
-                    throw new org.ballerinalang.core.util.exceptions.BLangRuntimeException(t.getMessage());
```

Comment:
Do we need to catch BLangRuntimeException and create new BLangRuntimeException?

Reply:
removed with the latest commit